### ISSUE TITLE
When changing custom project user directory all my editor settings disappeared?

### ISSUE THREAD

**Issue by Synzorasize** [godotengine/godot#47800]
**Godot version:**
3.2.3

**OS/device including version:**
Windows 10

**Issue description:**
I have no idea what happened exactly... I just changed the custom directory for my ongoing game since I didn't want it to be in the Godot folder (if people playing my games don't know what Godot is or think the file path is too complex, just so you know).
Then I decided to quit to project list because something was glitching out with my project (don't know if it might be another bug, but it had to do with StyleboxTextures turning into StyleboxFlats). However, when I got to the project list everything was reset, including the editor theme, projects, default project folder, etc. The editor layouts are still there, though. I did open the editor settings folder through the Godot engine only once, but that shouldn't have triggered a reset, I'm pretty sure.

**Steps to reproduce:**
I don't know exactly what happened, but I was changing the custom project directory directly to a folder in the AppData folder. Then I quit to project list.

**Minimal reproduction project:**

**Comment by Calinou** [comment]
> **OS/device including version:**

Please include this information.

Can you try to reproduce this a second time with another project? The fact that you lost your editor settings may be a concidence.

**Comment by Synzorasize** [comment]
Okay, I added the OS to the bug report. I tried this with a new project but nothing happened, really. One thing I forgot to mention originally is that I tested the new folder by having my game save to the new user:// folder. I also dragged and dropped my save files into that new folder before having the game save it. I suspect the other bug I mentioned which is why I quit to the project list might be because of cache (which is what I thought, I don't really know the inner workings of Godot resource-saving), so I was trying to clear it somehow by going to the project list. I don't know if that would also be a factor in what happened here, though, but it could be a possibility since it has to do with the file system.
I noticed that all my projects were back when I set the default project folder and autoscan project folder back to my Godot files folder, but this is probably normal.
